main.py

```
@@ -1,157 +1,100 @@
+        sys.stderr = self
+
+        return layout
+
+    def write(self, s):
+        self.log_output.text += s
+        self.log_output.cursor = (0, len(self.log_output.text))
+
+    def flush(self):
+        pass
+
+    def run_search_thread(self, instance):
```

Ruff (0.14.8)

Unused method argument: `instance`

(ARG002)